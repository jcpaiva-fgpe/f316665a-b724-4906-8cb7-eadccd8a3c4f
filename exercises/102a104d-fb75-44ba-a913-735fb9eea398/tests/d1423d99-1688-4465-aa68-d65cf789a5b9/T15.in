64{(50[o-29a]+a)-42{a-(s-22[v+41i])-s+{[c-22{f-b}]+[k-13(52(n-52b)-l+d-h)+92e]}}}+84{m+(63r+x-90o)+92u}
