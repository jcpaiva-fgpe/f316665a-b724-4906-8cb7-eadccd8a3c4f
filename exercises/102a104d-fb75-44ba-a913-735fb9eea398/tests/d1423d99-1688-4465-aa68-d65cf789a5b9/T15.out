[] 6 12
() 3 15
[] 27 33
() 22 34
{} 44 48
[] 39 49
() 59 65
() 56 72
[] 51 77
{} 38 78
{} 19 79
{} 2 80
() 87 97
{} 84 102
